### ISSUE TITLE
Use limit=false by default in show?

### ISSUE THREAD

**Issue by nalimilan** [JuliaData/DataFrames.jl#1631]
The `show` method for `Array` defaults to `:limit => false` when the I/O stream/context doesn't set the property (notably `stdout`). Maybe we should do the same for consistency. See discussion at https://github.com/JuliaData/DataFrames.jl/pull/1627.

**Comment by bkamins** [comment]
@ronisbr - is this still an issue with PrettyTables.jl?

**Comment by ronisbr** [comment]
Yes, the behavior is the same as before. If I use `show` without an io, then the output gets cropped by default.

**Comment by bkamins** [comment]
Then do you think it is worth to keep it open? (or maybe if it makes sense in general - maybe move the issue to PrettyTables.jl?)

**Comment by nalimilan** [comment]
One advantage of having `show(df)` not crop output is that people would easily have access to two different output formats: `df` at the REPL gives the cropped output, and `show(df)` would give the full output (instead of having to use e.g. `show(df, allrows=true)`. But the output isn't readable when columns don't fit in the screen, so that wouldn't be very useful. Maybe we could use `allrows=true` but `allcols=false` by default though.

TBH I'm not sure why arrays behave like they do. The result is generally unreadable even for small ones.

**Comment by bkamins** [comment]
I would just pick what we believe is useful. Being super consistent with Base here is not a top priority I think.

**Comment by nalimilan** [comment]
But what is useful? :-) Maybe make a poll in #data?

**Comment by bkamins** [comment]
sure why not. Could you do it please? (as I believe you know exactly which case is important to compare)

**Comment by ronisbr** [comment]
> Then do you think it is worth to keep it open? (or maybe if it makes sense in general - maybe move the issue to PrettyTables.jl?)

This cannot be fixed inside PrettyTables.jl. It is related to the default configurations of `allrows` and `allcols` depending on how `show` is called.

**Comment by bkamins** [comment]
OK - so let us keep it open here.
